website/content/v1.8/talos-guides/install/cloud-platforms/aws.md

```diff
@@ -408,29 +408,29 @@ talosctl config nodes $(aws ec2 describe-instances \
 Bootstrap `etcd`:
 
 ```bash
-talosctl bootstrap
```

Comment:
I think you missed the step on the link 386 (idk if the docs are not clear on that).

Reply:
Your right. Thanks